cmd/feature.go

```diff
@@ -7,12 +7,50 @@ import (
 	"strings"
 
 	"github.com/cozy/cozy-stack/client/request"
+	"github.com/cozy/cozy-stack/pkg/consts"
 	"github.com/spf13/cobra"
 )
 
 var featureCmdGroup = &cobra.Command{
-	Use:   "feature <command>",
-	Short: "Manage the feature flags",
+	Use:     "features <command>",
+	Aliases: []string{"feature"},
+	Short:   "Manage the feature flags",
+}
+
+var featureShowCmd = &cobra.Command{
+	Use:   "show",
+	Short: `Display the computed feature flags for an instance`,
+	Long: `
+cozy-stack feature show displays the feature flags that are shown by apps.
```

Comment:
probably we will someday need a `--source` or `--details` flag to get the whole computed flags sources like you have in the endpoint;